packages/frontend-2/components/settings/Dialog.vue

```diff
@@ -87,7 +109,10 @@ type MenuItem = {
 
 const { activeUser: user } = useActiveUser()
 const breakpoints = useBreakpoints(TailwindBreakpoints)
+const { result: workspaceResult } = useQuery(settingsSidebarWorkspacesQuery)
```

Comment:
You need to introduce and start using a similar composable to `useIsAutomateModuleEnabled()` that check the workspaces feature flag.

Cause this query should not even be firing with that FF off - otherwise the server is gonna throw an error. So the workspace settings UI & also all the relevant GQL operations should be blocked by the feature flag.

Reply:
Good point; didn't know they would throw an error when not available. I've adjusted this to use a feature flag, following the example of automate